### PR TITLE
modified file - _posts/0000-01-23-manas-mukherjee.md

### PR DESCRIPTION
**Pull request by manas-mukherjee**
Hello, 
Creating the first pull request. 
Thanks

### CODE REVIEW

File: _posts/0000-01-23-manas-mukherjee.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![angry-cat](https://cloud.githubusercontent.com/assets/16547949/25400569/064d7bc6-29c1-11e7-90dc-fef44edd9f0a.jpg)
 {: .center}
 
-Caption here
+Hi! I'm Manas.
```
Comment (dhanyaramesh) [inline _posts/0000-01-23-manas-mukherjee.md]: Nice 👍

**Review by spoondla1 — DISMISSED** [review]
hi

**Review by dhanyaramesh — CHANGES_REQUESTED** [review]
Would you want to try giving another caption?
Just for the sake of it :P 
ha ha ha

**Review by dhanyaramesh — APPROVED** [review]
LGTM :)

**Comment by manas-mukherjee** [comment]
Merging branch to master. Thanks for approving

**Pull request merged**